test/python/circuit/library/test_multipliers.py

```diff
@@ -139,10 +139,13 @@ def test_modular_cumulative_multiplier_custom_adder(self):
             _ = HRSCumulativeMultiplier(3, 3, adder=VBERippleCarryAdder(3))
 
     def test_plugins(self):
-        """Test setting the HLS plugins for the modular adder."""
+        """Test setting HLS plugins for the multiplier."""
 
-        # all gates with the plugins we check, including an expected operation
-        plugins = [("cumulative_h18", "ch"), ("qft_r17", "mcphase")]
+        # For each pliugin, we check the presence of an expected operation after
```

Comment:
pliugin --> plugin

Reply:
thanks for spotting this :) fixed in [29cd6d7](https://github.com/Qiskit/qiskit/pull/13916/commits/29cd6d776fffcdb2601419de69595f6b31daffb3)